md/release-notes.md

```diff
@@ -62,7 +82,6 @@ Same thing if you have a logout link in a custom page that does not pass a `logi
 If you use Bonita layout version 5 or a customized version of it in your applications, make sure you upgrade to version 6 when migrating. Otherwise, the logout button will not redirect to the login page when clicked.
 
 #### Search keys
```

Comment:
Isn't that already part of the Reinforce Open Source DNA chapter at the top of the page?